util/package.json

```diff
@@ -4,10 +4,12 @@
   "description": "Script to help build required JSON from Excel",
   "main": "index.js",
   "scripts": {
-    "test": "echo \"Error: no test specified\" && exit 1"
+    "test": "echo \"Error: no test specified\" && exit 1",
+    "convert": "node convert_to_json.js && node json_to_data_dir.js"
```

Comment:
What is this new `convert` script? What does it do? Update README file as well on it's usage.

Reply:
Split the task of converting to JSON and then doing transformations on top of that into different scripts.